__tests__/app.test.js

```diff
@@ -94,8 +105,8 @@ describe("NC_Games API", () => {
       const reqBody = { invalid: 10 };
       return request(app)
         .patch("/api/reviews/12345")
-        .send(body)
-        .expect(400)
+        .send(reqBody)
+        .expect(404)
         .then(({ body }) => {
```

Comment:
Just missing a test:

- [ ] status 400 - they try and patch with something that isn't a number `{inc_votes: 'not a number'}`